app/classifier/tasks/survey/editor.cjsx

```diff
@@ -327,11 +322,10 @@ module.exports = createReactClass
           <select
             name="#{@props.taskPrefix}.thumbnails"
             onChange={handleInputChange.bind @props.workflow}
-            value={if @props.task.thumbnails then @props.task.thumbnails else if @props.task.alwaysShowThumbnails then "show" else "default"}
+            value={if @props.task.thumbnails then @props.task.thumbnails else "default"}
           >
             <option value="default">Default</option>
             <option value="hide">Hide</option>
-            <option value="show">Show</option>
```

Comment:
Note about backwards-compatibility: if we want to correctly display the "Show" option for _existing projects that still have thumbnails = 'show',_ then we can change the code here to look something like...

```
{if @props.task.thumbnails is "show" then
  <option value="show">Show</option>
}
```

This means the "show" option correctly displays if and only if the Survey Task already has it set that way. As soon as the Survey Task changes to default or hide, the "show" option disappears.